test/e2e-tests.sh

```diff
@@ -29,6 +29,25 @@
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
 source $(dirname $0)/e2e-common.sh
 
+# Helper functions.
+function dump_app_logs() {
```

Comment:
This is nice. We should move this to the helper script so every repo can use it.

Reply:
I agree completely

Reply:
Filed https://github.com/knative/test-infra/issues/252 so we don't forget. :)